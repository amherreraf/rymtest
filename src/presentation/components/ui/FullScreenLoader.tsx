import {StyleSheet, View, ActivityIndicator} from 'react-native';

export const FullScreenLoader = () => {
  return (
    <View style={styles.all}>
      <ActivityIndicator size={50} />
    </View>
  );
};

const styles = StyleSheet.create({
  all: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
