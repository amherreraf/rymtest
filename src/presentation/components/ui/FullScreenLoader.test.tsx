import React from 'react';
import renderer from 'react-test-renderer';
import {FullScreenLoader} from './FullScreenLoader';

describe('tests on Input', () => {
  it('without props', () => {
    const tree = renderer.create(<FullScreenLoader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
