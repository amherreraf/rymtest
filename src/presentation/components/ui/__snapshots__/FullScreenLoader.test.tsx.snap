// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on Input without props 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    size={50}
  />
</View>
`;
